Makefile_packaging.mk

```diff
@@ -260,13 +260,25 @@ enabled=1\n" >> /etc/mock/default.cfg;
 	fi
 	mock $(MOCK_OPTIONS) $(RPM_BUILD_OPTIONS) $<
 else ifeq ($(ID_LIKE),debian)
-ifneq ($(ubuntu1804_REPOS),"")
+ifneq ($(DAOS_STACK_REPO_SUPPORT),"")
+ifneq ($(DAOS_STACK_REPO_UBUNTU_18_04_LIST),"")
+ubuntu1804_REPOS := $(shell curl $(DAOS_STACK_REPO_SUPPORT)$(DAOS_STACK_REPO_UBUNTU_18_04_LIST))
```

Comment:
Out of interest, what's the content of fetching this URL?

Reply:
Digging, I see it's:

```
deb [arch=amd64] https://repo.dc.hpdd.intel.com/repository/daos-stack-ubuntu-18.04-x86_64-stable-local/ bionic main
```

Is there a benefit to fetching that rather than just having it in a per-site global variable?

Reply:
The list file was setup because Nexus does not support APT group repositories.
This was to allow installing one list file and a set of GPG keys on the CI or target build system.

It was setup before the Ubuntu pbuilder requirements were known.
.
As long the list file has only one repository in it and no comments than this works.

For a more general solution we need either multiple list files restricted to one repository each, or we need to have some way for the Makefile to read and parse the repository file into the format that pbuilder wants for multiple repositories.